tests/test_providers/test_person.py

```diff
@@ -88,17 +85,22 @@ def test_username(self, _person, template):
             'l_d': r'^[a-z]+_[0-9]+$',
             'ld': r'^[a-z]+[0-9]+$',
             'default': r'^[a-z]+\.[0-9]+',
+            None: r'^[A-Za-z]{2,}[\.\-\_]?[0-9]+$',
```

Comment:
@ceccoemi Are you sure that tests cover all the templates?

Reply:
Yes I have spent a lot of time in building that regex and I am pretty sure that it covers all the templates that are used when `template=None`.

I want to clarify that the regex do not cover all the possible usernames with all the possible templates, but it refers to `(‘U_d’, ‘U.d’, ‘U-d’, ‘UU-d’, ‘UU.d’, ‘UU_d’, ‘ld’, ‘l-d’, ‘Ud’, ‘l.d’, ‘l_d’)`, that are the templates used when `template=None`.